src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -12,8 +12,32 @@ import { EmptyStateCdMaterial } from './EmptyStateCdMaterial';
 import { getCDModalHeader, CDButtonLabelMap } from './config';
 import { CDModalTab } from '../../service';
 import GitCommitInfoGeneric from '../../../common/GitCommitInfoGeneric';
+import { getModuleInfo } from '../../../v2/devtronStackManager/DevtronStackManager.service';
+import { ModuleNameMap } from '../../../../config';
+import { ModuleStatus } from '../../../v2/devtronStackManager/DevtronStackManager.type';
 
-export class CDMaterial extends Component<CDMaterialProps> {
+export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
+  constructor(props: CDMaterialProps) {
+    super(props)
+    this.state = {
+      isSecurityModuleInstalled: false
+    }
+  }
+
+  componentDidMount() {
+    this.getSecurityModuleStatus()
+  }
+
+  async getSecurityModuleStatus(): Promise<void> {
+    try {
+        const { result } = await getModuleInfo(ModuleNameMap.SECURITY)
+        if (result?.status === ModuleStatus.INSTALLED) {
+            this.setState((prevState) => ({
+                isSecurityModuleInstalled: true,
+            }))
```

Comment:
Can be changed to `this.setState({ isSecurityModuleInstalled: true })`?

Reply:
done